demo/px-data-table-highlight-demo.html

```diff
@@ -34,7 +34,22 @@
 <!-- Demo DOM module -->
 <dom-module id="px-data-table-highlight-demo">
   <template>
-
+    <style>
+        .message-banner {
```

Comment:
Indentation on lines 38-51, 62-71 should use 2 spaces for indentation (annoying that prettier doesn't fix HTML files!)